hedera-node/src/main/java/com/hedera/services/state/merkle/MerkleNetworkContext.java

```diff
@@ -200,9 +192,9 @@ public void deserialize(SerializableDataInputStream in, int version) throws IOEx
 
 			int numCongestionStarts = in.readInt();
 			if (numCongestionStarts > 0) {
-				congestionLevelStarts = new RichInstant[numCongestionStarts];
+				congestionLevelStarts = new Instant[numCongestionStarts];
 				for (int i = 0; i < numCongestionStarts; i++) {
-					congestionLevelStarts[i] = serdes.readNullableInstant(in);
+					congestionLevelStarts[i] = serdes.readNullableInstant(in).toJava(); // ifPresent...
```

Comment:
This must be fixed! `serdes.readNullableInstant(in)` will either return a `null` or a real `RichInstant`. Have to decide what to do with a `null`!

Reply:
This is fixed.